solidity/contracts/converter/types/liquidity-pool-v1/LiquidityPoolV1Converter.sol

```diff
@@ -155,6 +189,69 @@ contract LiquidityPoolV1Converter is LiquidityPoolConverter {
         return amount;
     }
 
+    /**
+      * @dev returns the recent average rate of 1 `_token` in the other reserve token units
+      * note that the rate can only be queried for reserves in a standard pool
+      *
+      * @param _token   token to get the rate for
+      * @return recent average rate between the reserves (numerator)
+      * @return recent average rate between the reserves (denominator)
+    */
+    function recentAverageRate(IERC20Token _token) external view returns (uint256, uint256) {
+        // if the pool isn't standard, return 0, 0
+        if (!isStandardPool) {

```

Comment:
Do we really want to do a storage read just in order to return this "invalid-anyway" value?
Why don't we just add a comment in the Natspec documentation above, stating that this function is designated only for standard pools?

Reply:
for the external version, I think it's better, otherwise you get undefined behavior

Reply:
@yudilevi : So perhaps `require(isStandardPool, "ERR_MSG");` is better than returning rate = 0/0 (which is also pretty undefined).

Reply:
done